cypress/integration/top-above-nav.spec.js

```diff
@@ -0,0 +1,64 @@
+/// <reference types="cypress" />
+
+import { getStage, getTestUrl } from '../lib/util';
+import { getIframeBody } from '../lib/iframe';
+
+// Don't fail tests when uncaught exceptions occur
+// This is because scripts loaded on the page and unrelated to these tests can cause this
+Cypress.on('uncaught:exception', (err, runnable) => {
+	return false;
+});
+
+const stage = getStage();
+
+const pages = [
+	{
+		path: getTestUrl(stage, ''),
+		adTest: 'fixed-puppies',
+	},
+	{
+		path: getTestUrl(
+			stage,
+			'/politics/2022/feb/10/keir-starmer-says-stop-the-war-coalition-gives-help-to-authoritarians-like-putin',
+			{ isDcr: true },
+		),
+		adTest: 'fixed-puppies',
+	},
+	{
+		path: getTestUrl(
+			stage,
+			'/politics/live/2022/jan/31/uk-politics-live-omicron-nhs-workers-coronavirus-vaccines-no-10-sue-gray-report',
+			{ isDcr: false },
+		),
+		adTest: 'fixed-puppies',
+	},
+	{
+		path: getTestUrl(
+			stage,
+			'/sport/2022/feb/10/team-gb-winter-olympic-struggles-go-on-with-problems-for-skeleton-crew',
+			{ isDcr: true },
+		),
+		adTest: 'fixed-puppies',
+	},
+];
+
+describe('top-above-nav on pages', () => {
+	pages.forEach(({ path, adTest }) => {
+		it(`Test ${path} has top-above-nav slot and iframe`, () => {
+			cy.visit(`${path}?adtest=${adTest}`);
+
+			// Click "Yes, I'm happy" on the sourcepoint banner to obtain consent
+			getIframeBody('sp_message_iframe_597005')
+				.find('.btn-primary', { timeout: 10_000 })
```

Comment:
Just wondering if the default timeouts don't work for us should we configure them globally?

https://docs.cypress.io/guides/references/configuration#Timeouts

Reply:
Nice, have updated.

I've also bumped this default timeout to 15s since finding elements very occasionally takes longer than 10s.